src/Microsoft.Extensions.RuntimeEnvironment.Sources/RuntimeEnvironment.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+
+namespace Microsoft.Extensions.Internal
+{
+    internal class RuntimeEnvironment
+    {
+        public RuntimeEnvironment()
+        {
+            OperatingSystemVersion = Native.PlatformApis.GetOSVersion();
+
+            RuntimeType = GetRuntimeType();
+        }
+
+        public string OperatingSystemVersion { get; }
```

Comment:
Could you list the places where we use this? If it's just Testing and runtime information, we could drop this entirely.